Documentation and provider update

sachasmart opened the pull request:
- Small update on documentation to use `url` instead of host (no longer the correct attributes 🤷 )
- Small update to the error message to display the `url` that was passed to the `provider`

marshallford commented:
@sachasmart Thanks for the PR! I'll take a look this week when I have a bit of time. In the meantime could you look into regenerating the docs? Thanks again.

sachasmart commented:
@marshallford No problem! Looks like a promising provider to manage pfSense in my homelab. 

I believe I made a slightly hasty PR and just updated the `index.md` on the initial push. With your latest comment, I've updated the `examples/provider/provider.tf` and generated the docs, which seems now seems to accurately reflect the intentions of the initial PR.
